modules/installing-ocp-agent-tui.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-tui_{context}"]
+= Using the `agent-tui` application to check for network configuration errors
```

Comment:
Rather than putting the accent on the tool (`agent-tui`), and on the network configuration, I feel it could sound better to introduce the feature, which is the ability to verify if the current host can successfully pull the configured release image or not: from the point of view of an user, this is a specific agent capability that could be used to act in case of error

Reply:
Also, it could be worth to properly highlight, from the very beginning, that in case of error the installation workflow will be halted until the user will take control (and stop the console app to signal the intention to proceed)